### PR TITLE
feat(ext-update): update ext-image files in gdc-nas

### PR DESCRIPTION
**Pull request by petr-klemsinsky**
Use new ext-image update implementation which is based on Github actions

JIRA: INFRA-3327

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/210/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/216/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/212/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/218/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/215/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/220/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/219/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/223/ : SUCCESS in 4s

File: .github/workflows/post-merge-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+# (C) 2024 GoodData Corporation
+name: Update Tiger ext image
+"on":
+  push:
+    branches: ["develop", "release", "hotfix/tiger", "hotfix/panther", "fast_track"]
```
Comment (tuan-tran-thanh) [inline .github/workflows/post-merge-pipeline.yml:6]: it is `master`, not `develop`

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/220/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/224/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/221/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/225/ : SUCCESS in 3s

**Review by tuan-tran-thanh — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-public-demos-gate-PR91

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**